bowler/tool.py

```diff
@@ -239,10 +255,22 @@ def refactor(self, items: Sequence[str], *a, **k) -> None:
 
         while True:
             try:
-                filename, hunks = self.results.get_nowait()
-                self.log_debug(f"results: got {len(hunks)} hunks for {filename}")
+                filename, hunks, exc = self.results.get_nowait()
                 results_count += 1
-                self.process_hunks(filename, hunks)
+
+                if isinstance(exc, BowlerException):
```

Comment:
Because on line 216 above you put non-`BowlerException` exceptions in there too, maybe `if exc` here, and move the isinstance down to where you deal with hunks on line 267?